Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/TorrentDownloader.cs

```diff
@@ -67,47 +105,25 @@ public void Download(CancellationToken cancellationToken)
 
                 _logger.LogTrace("download dir  = " + DestinationDirectoryPath);
 
-                using (var torrentClient = new TorrentClient(new UnityTorrentClientProcessStartInfoProvider()))
-                {
-                    torrentClient.AddTorrent(_torrentFilePath, DestinationDirectoryPath, cancellationToken);
-
-                    var timeoutWatch = new Stopwatch();
-                    timeoutWatch.Start();
-
-                    TorrentStatus status = GetAndCheckTorrentStatus(torrentClient, cancellationToken);
-                    double initialProgress = status.Progress;
-                    _logger.LogTrace("initialProgress = " + status.Progress);
-                    var waitHandle = new AutoResetEvent(false);
+                IRequestRetryStrategy retryStrategy = new RetryStrategy(3);
```

Comment:
Why only three retries?

Reply:
I picked this value completely arbitrarily

Reply:
Ok, it should be fine.